### ISSUE TITLE
5.3.1 Symlink Error after composer update

### ISSUE THREAD

**Issue by maciejnecki** [neos/neos-ui#2790]
### Description

Source directory could not be symlinked in 5.3.1 branch after updating. Public resources can't be published. 

### Steps to Reproduce

1. Composer Update to 5.3.1
2. Refresh Neos ui or attempt any flow command

#### Expected behavior

Flow operating normally with no exceptions thrown. 

#### Actual behavior

https://pastebin.com/axq5eigr

### Affected Versions

Neos: 5.1.10

UI: 5.3.1

Workaround was to revert to 5.2.3

You can get this information by running `composer show` or using the package management module within Neos.
The UI version can also be found inside the left sidebar after version 1.0.3.

**Comment by markusguenther** [comment]
Hello and thanks for reporting that @maciejnecki,

I personally can not reproduce that. Did you try to remove the caches after the composer update.

For instance with ` rm -rf Data/Temporary/`
Would be interesting what kind of a setup you have, when the issue still exists after hard cache removal.

**Comment by maciejnecki** [comment]
Hi @markusguenther, apologies for getting back to you on this late. 

I will close this issue as after some more testing it seems to be not directly related. 

---

The synopsis is that I had a Neos installation running locally and I have run `composer update` getting the latest version. Then I have followed the standard update steps of clearing cache, publishing resources, etc., and the update did work. However, following some changes to the source code and pushing them to git, I have run another update where only my new changes were integrated for testing. At that point Flow has refused to cooperate with the error I have described, not matter what command I had issued and not matter the cache cleaning process. 

I have then downgraded neos/ui and that solved the symlinking issue until next code changes and publish process. 

I have finally solved this ordeal by deleting the whole installation and downloading a fresh Neos install that I have then linked with may site package from git and existing database. Now everything is working but I am afraid that I will never know what was the cause of that issue. 

---

The local install was running on macOS 10.15.6 with latest mariaDB through brew. 
Only other observation was that following composer update, I was getting a different error with some of Neos dependencies that I have narrowed to php@7.3 incompatibility, update to 7.4 has fixed those but also that's where the symlinking issues occurred. 

As a side note In my case it seems now that I can not run latest Neos without php@7.4 despite documentation requiring php@7 or newer. Perhaps something to look at? 

Thanks for your help!
